README.md

```diff
@@ -0,0 +1,244 @@
+# tox-pipenv2
+
+A reimplementation of [`tox-dev/tox-pipenv`](https://github.com/tox-dev/tox-pipenv) to address
+existing issues and rough edges encountered in a "strict-pinning" workflow.
+
+
+[![tox-pipenv on PyPI](https://img.shields.io/pypi/v/tox-pipenv2.svg)](https://pypi.python.org/pypi/tox-pipenv2)
+
+[![Test Package Compatibility](https://github.com/tox-dev/tox-pipenv/workflows/Test%20Package%20Compatibility/badge.svg)](https://github.com/tox-dev/tox-pipenv/actions)
+
+![PyPI downloads](https://img.shields.io/pypi/dw/tox-pipenv2)
+
+A tox plugin to replace the default use of pip with Pipenv.
+
+* Use `Pipfile` with multiple version of Python
+* Use `tox` to deploy strict-pinned dependencies from `Pipfile.lock.{envname}`
+  * Update lock files for multiple environments
+
+# Usage
+
+## `tox.ini`
+
+Add `tox-pipenv` to the top level `requires` list. Tox will automatically
+provision an environment with the given requirements.
+
+    [tox]
+    requires = tox-pipenv
+
+    [testenv]
+    command = {posargs}
+
+## `{toxinidir}/Pipfile`
+
+Use `pipenv` to create a `Pipfile` in the project directory, next to `tox.ini`.
+
+For tox environments without `deps`, this plugin will automatically use
+`pipenv install` to apply the `Pipfile` to the tox virtualenv.
+
+Environments that specify any `deps` will be _ignored_ by this plugin, as
+if `skip_pipenv = true` were set.
+
+### `{toxinidir}/Pipfile.lock.{envname}`
+
+If a lock file exists that matches the current tox environment name, this
+plugin will use `pipenv sync` to apply the lockfile into the tox virtualenv.
+
+# Regenerating the lock file
+
+When using lock files ("strict pinning"), the per-environment
+`Pipfile.lock.{envname}` must be regenerated whenever `Pipfile` changes or when
+updating dependency pins.
+
+```
+tox --recreate --pipenv-lock
```

Comment:
`--pipenv-lock` is now `--pipenv-update` and works like `pipenv update`, except instead of updating `Pipfile.lock`, it updates `Pipfile.lock.{envname}`.

(`--pipenv-lock` is removed entirely)